packages/devkit/src/utils/match-path-with-template-string.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { matchPathWithTemplateString } from './match-path-with-template-string';
+
+describe('offsetFromRoot', () => {
```

Comment:
If we keep this function, make sure to fix the describe name

```suggestion
describe('matchPathWithTemplateString', () => {
```

Reply:
ooops!!!! hehehe